### ISSUE TITLE
#nosec comment in code to exclude results

### ISSUE THREAD

**Issue by jonny-wg2** [Checkmarx/kics#3843]
**Is your feature request related to a problem? Please describe.**
I have the use case where I would like to exclude results from kics scans. Further more I would like developers to easily exclude kics results without having to update the ci kics configuration.

**Describe the solution you'd like**
The ability to set a flag when running kics to exclude results when there is a comment above the code. Default could be #nosec and then ability to also set your own. 

**Describe alternatives you've considered**
Kics has the ability to exclude a rule but I would like to keep a rule but exclude a result. With code being the source of truth, having this done in code would be awesome. 

**Additional context**
Gosec has a similar approach. https://github.com/securego/gosec

**Comment by felipe-avelar** [comment]
@jonny-wg2 Hello, first of all I hope you are doing well. 🙂 
Reading your request, the flag `exclude-paths` solve your situation, no?
Also, there is a problem with the comment approach... KICS supports multiple files extensions and some of this extensions doesn't allow comments, like JSON, this is why KICS has `exclude-paths` flag.

**Comment by jonny-wg2** [comment]
Hi @felipe-avelar ! :D 

To set some background this is the tree output
```
│   ├── containers
│   │   ├── image-1
│   │   │   ├── BUILD
│   │   │   ├── Dockerfile
│   │   │   ├── README.md
│   │   │   └── docker-compose.yml
│   │   ├── image-2
│   │   │   ├── BUILD
│   │   │   ├── Dockerfile
│   │   │   └── README.md
```
I looked at `exclude-path`, however I would not want to exclude all of `containers/image-1` for every Dockerfule rule, but just exclude `containers/image-1` from one rule.

Also, with this deployed as part of our ci, having developers update the ci is not that intuitive, but if we provide a message saying, update false positives with `#nosec` then that could be pretty neat. 

With regards to `json` not supporting comments, perhaps we just don't support json with this feature? 

But if you have another solution to my problem happy to hear.

**Comment by felipe-avelar** [comment]
@jonny-wg2 now I see your problem. I think your suggestion is great and we will be working on a solution to this problem. 🙂

**Comment by rogeriopeixotocx** [comment]
We should start by adding the ability to ignore files and queries with a comment near the header of the file.

**Comment by felipe-avelar** [comment]
@jonny-wg2 I'm happy to say this feature is available in v1.4.1 😃

**Comment by jonny-wg2** [comment]
Wooohoooo You da 💣 @felipe-avelar . So can't wait to update!!
